docs/software/requesting-tokens.md

```diff
@@ -26,104 +26,139 @@ and refresh tokens from OpenID Connect token providers.
+                Issuer [https://iam-test.indigo-datacloud.eu/]: https://wlcg.cloud.cnaf.infn.it/
 
-        | **Capability**   | **Scope**                     |
-        |------------------|-------------------------------|
-        | HTCondor `READ`  | `compute.read`                |
-        | HTCondor `WRITE` | `compute.modify compute.cancel compute.create` |
-        | XRootD read      | `read:/`                      |
-        | XRootD write     | `write:/`                     |
+        1. Request `wlcg`, `offline_access`, and other scopes for the capabilities that you need:
 
-        For example, to request HTCondor `READ` and `WRITE` access, specify the following scopes:
+            | **Capability**   | **Scope**                     |
+            |------------------|-------------------------------|
+            | HTCondor `READ`  | `compute.read`                |
+            | HTCondor `WRITE` | `compute.modify compute.cancel compute.create` |
+            | XRootD read      | `read:/`                      |
+            | XRootD write     | `write:/`                     |
 
-            This issuer supports the following scopes: openid profile email address phone offline_access wlcg iam wlcg.groups
-            Space delimited list of scopes or 'max' [openid profile offline_access]: wlcg offline_access compute.read compute.modify compute.cancel compute.create
+            For example, to request HTCondor `READ` and `WRITE` access, specify the following scopes:
+
+                This issuer supports the following scopes: openid profile email address phone offline_access wlcg iam wlcg.groups
+                Space delimited list of scopes or 'max' [openid profile offline_access]: wlcg offline_access compute.read compute.modify compute.cancel compute.create
     
-        Note that, prior to HTCondor 8.9.7, the server also needed `condor:/ALLOW` in all cases.
+            Note that, prior to HTCondor 8.9.7, the server also needed `condor:/ALLOW` in all cases.
 
-    1. When prompted, open <https://wlcg.cloud.cnaf.infn.it/device> in a browser, enter the code provided by `oidc-gen`,
+        1. When prompted, open <https://wlcg.cloud.cnaf.infn.it/device> in a browser, enter the code provided by `oidc-gen`,
        and click "Submit".
 
-    1. On the next page, verify the scopes and client profile name, and click "Authorize".
+        1. On the next page, verify the scopes and client profile name, and click "Authorize".
 
-    1. Enter a password to encrypt your local client profile.
+        1. Enter a password to encrypt your local client profile.
        You'll need to remember this if you want to re-use this profile in subsequent sessions.
 
-1. Request a token using the client name that you used above with `oidc-gen`:
+1. Request a token using the client profile that you used above with `oidc-gen`:
+
+    !!! note
+        You must first [register a new profile](#using-a-container)
```

Comment:
When reorganizing, make sure to point to the corresponding profile registration section